src/api/order/email/domain/order-email-domain.v6.service.js

```diff
@@ -0,0 +1,44 @@
+angular.module('ovh-api-services').service('OvhApiOrderDomainV6', ($resource, $cacheFactory) => {
+  // Cache to invalidate
+  const queryCache = $cacheFactory('OvhApiOrderDomainV6Query');
+  const cache = $cacheFactory('OvhApiOrderDomainV6');
+
+  const interceptor = {
+    response(response) {
+      orderEmailDomain.resetQueryCache();
+      return response.data;
+    },
+  };
+
+  const orderEmailDomain = $resource('/order/email/domain/:domain', {
+    domain: '@domain',
+  }, {
+    query: { method: 'GET', cache: queryCache, isArray: true },
+    getDuration: {
```

Comment:
```suggestion
    getOption: {
```
would be better as tehre is no `/duration` in the route and you have to pass an `option` parameter